framework/base/CSecurityManager.php

```diff
@@ -332,7 +332,7 @@ public function computeHMAC($data,$key=null,$hashAlgorithm=null)
 			$hashAlgorithm=$this->hashAlgorithm;
 
 		if(function_exists('hash_hmac'))
-			return hash_hmac($hashAlgorithm,$data,$key);
+			return self::hash_hmac_fallback($hashAlgorithm,$data,$key);
```

Comment:
What are use-cases for using `computeHmac()` with sha1 or md5?

Reply:
Not sure what you mean by the question.  I encountered an issue when not using 'sha1' or 'md5' because hash_hmac stopped supporting it.